src/arm/96boards.c

```diff
@@ -53,7 +60,6 @@ int hikey_ls_gpio_pins[MRAA_96BOARDS_LS_GPIO_COUNT] = {
     488, 489, 490, 491, 492, 415, 463, 495, 426, 433, 427, 434,
 };
 
-
```

Comment:
This is another surplus styling change I mentioned in the previous review. Please revert this.

Reply:
done